Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -1474,6 +1474,7 @@ def _split_workspace(self, raw_ws_name, split_ws_name):
                              SplitterWorkspace=split_ws_name, GroupWorkspaces=True, RelativeTime=is_relative_time)
         else:
             # split with information table
+            self.log().notice('[DB...BAT] Raw Name: {0}.  Base Name: {1}'.format(raw_ws_name, base_name))
```

Comment:
It was for debugging.  I forgot to delete it afterwards.  Meanwhile, I cleaned up some other debugging outputs assigned to wrong logging level.  Thanks!